src/CoolPropLib.cpp

```diff
@@ -101,99 +101,101 @@ double convert_from_SI_to_kSI(long iInput, double value)
 }
 
 EXPORT_CODE long CONVENTION redirect_stdout(const char* file){
+    fpu_reset_guard guard;
     FILE *fp = freopen(file, "a+", stdout);
-    reset_fpu();
```

Comment:
No, that should be fine to remove the reset_fpu() call

On Sun, Jun 5, 2016 at 1:10 AM, Mike Kaganski <notifications@github.com>
wrote:

> In src/CoolPropLib.cpp
> <https://github.com/CoolProp/CoolProp/pull/1097#discussion_r65812374>:
>
> >      FILE *fp = freopen(file, "a+", stdout);
> > -    reset_fpu();
>
> Actually, I don't know why fpu resetting is required here. Does freopen()
> do floating-point math?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CoolProp/CoolProp/pull/1097/files/dbae9a1d8804d25845203523cc828f065814dba9#r65812374>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABxha6Mod6TQ4X7kSfVb-YPwGiCD1PgXks5qInZdgaJpZM4IuIEV>
> .
>